### PR TITLE
Fix/add descriptions

### PR DESCRIPTION
**Pull request by bdmogal**
Jira: [CDAP-13389](https://issues.cask.co/browse/CDAP-13389)

### CODE REVIEW

File: cdap-runtime-ext-dataproc/src/main/resources/gce-dataproc.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
           "widget-type": "textbox",
           "label": "Network",
           "name": "network",
+          "description": "Select the default network or any VPC network created in the specified project to use for creating clusters with this profile.",
```
Comment (albertshau) [inline cdap-runtime-ext-dataproc/src/main/resources/gce-dataproc.json]: Select the VPC network in the specified project. Putting 'default' here is confusing, as there is a network called 'default'

Comment (bdmogal) [reply cdap-runtime-ext-dataproc/src/main/resources/gce-dataproc.json]: fixed

**Comment by bdmogal** [comment]
@albertshau addressed comment. please take another look

**Review by albertshau — APPROVED** [review]
lgtm

**Comment by bdmogal** [comment]
Made sure the json is correct, and tested it in the UI. Merging since this does not involve any code changes.

**Pull request merged**